### DISCUSSION TITLE
[Time Series] Add Option to make positive and negative axis symetrical

### DISCUSSION THREAD

**Discussion by yurividal** [grafana/grafana#38130] [Dashboards, Panels & Visualizations]
When using auto for y-min and y-max, there could be an **option** to keep them always symetrical. It can be misleading when the +y axis is, for example, a lot larger than the -y axis,

![image](https://user-images.githubusercontent.com/8555749/126057143-3394580b-f1d5-444a-956e-ef60e334d953.png)
In this case, for example, if +y had automatically grown to 76Mib/s it would have been more intuitive for the viewer.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by baryluk** [comment]
Yes I agree. Network downlink + uplink, as well time synchronization offset (i.e. from ntp, chrony, and a like), are few of the best examples where one would like to have symmetric positive and negative extent in the graph.

**Comment by mave007** [comment]
I liked the way it was managed on the old graph. Each Y-axis can have it's own min and max detailed.

In the case of a butterfly network graph, I would like to use traffic going in on the Y-1 left axis (as a positive) and the traffic going out on the Y-2 axis (as a negative value). Then I would adjust min and max depending on each Y axis

**Comment by nmarrs** [comment]
This is most likely addressed by https://github.com/grafana/grafana/pull/52555

**Reply by baryluk** [reply]
Indeed. It looks like it solves this issue. Will test once I have a chance.

**Reply by baryluk** [reply]
Yes. This is fixed, and work nicely. Not only y can be made symmetric (centered zero), stacking separately negative and positive also works (different issue, but related).

Example

![image](https://user-images.githubusercontent.com/110322/224156256-bce46ef4-8897-4e1d-89bd-8f26966fdf2c.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
